src/main/java/mekanism/common/FuelHandler.java

```diff
@@ -56,7 +58,7 @@ public FuelGas(int duration, double energyDensity)
 		public FuelGas(IFuel bcFuel)
 		{
 			burnTicks = bcFuel.getTotalBurningTime() / Fluid.BUCKET_VOLUME;
-			energyPerTick = bcFuel.getPowerPerCycle() * general.FROM_RF;
+			energyPerTick = bcFuel.getPowerPerCycle() * MjAPI.MJ;
```

Comment:
Uhm.. I did read the value the wrong way, it should be a division of course. The conversion between Mekanism Joules and BC Joules, I don't know about it...

It's updated, I'll leave the confirmation of the values to Aidan to confirm ;)
Thanks for the input! If you see some more stuff, please let me know!